018_LeapYear.js

```diff
@@ -0,0 +1,12 @@
+// 18. Write a program that prints the next 20 leap years.
+
+const leapYear = (year) => {
```

Comment:
Your program should decide where to stop the loop. You  are calculating the 2100 by yourself, that is not the correct way